Content.Shared/_Shitmed/Surgery/SharedSurgerySystem.Steps.cs

```diff
@@ -923,7 +923,13 @@ private float GetSurgeryDuration(EntityUid surgeryStep, EntityUid user, EntityUi
             return 2f; // Shouldnt really happen but just a failsafe.
 
         var speed = toolSpeed;
-
+        if(TryComp<BuckleComponent>(target, out var buckleComp)) // Get buckle component from target.
+        {
+            if(TryComp<OperatingTableComponent>(buckleComp.BuckledTo, out var operatingTableComponent))  // If they are buckled to entity with operating table component
+            {
+                speed *= operatingTableComponent.SpeedModifier; // apply surgery speed modifier
+            }
+        }
```

Comment:
Chief, the brackets must go. They are without their families for too long. They need not be here for a simple one liner. The union must rise.
```suggestion
        if(TryComp<BuckleComponent>(target, out var buckleComp)) // Get buckle component from target.
            if(TryComp<OperatingTableComponent>(buckleComp.BuckledTo, out var operatingTableComponent))  // If they are buckled to entity with operating table component
                speed *= operatingTableComponent.SpeedModifier; // apply surgery speed modifier
```